ledger/acctupdates_test.go

```diff
@@ -1030,3 +1030,92 @@ func TestListCreatables(t *testing.T) {
 		expectedDbImage, numElementsPerSegement)
 	listAndCompareComb(t, au, expectedDbImage)
 }
+
+func TestIsWritingCatchpointFile(t *testing.T) {
```

Comment:
I think that this test is not very meaningful, but it's ok to have it for now.
( i.e. it's not wrong.. just overly dependent in the internal implementation )